openpaas-james/jmap/extensions/src/main/scala/com/linagora/openpaas/james/jmap/method/KeystoreSetMethod.scala

```diff
@@ -0,0 +1,80 @@
+package com.linagora.openpaas.james.jmap.method
+
+import com.google.inject.AbstractModule
+import com.google.inject.multibindings.{Multibinder, ProvidesIntoSet}
+import com.linagora.openpaas.james.jmap.json.KeystoreSerializer
+import com.linagora.openpaas.james.jmap.method.CapabilityIdentifier.LINAGORA_PGP
+import com.linagora.openpaas.james.jmap.model.{KeystoreSetRequest, KeystoreSetResponse}
+import eu.timepit.refined.auto._
+import org.apache.james.jmap.core.CapabilityIdentifier.{CapabilityIdentifier, JMAP_CORE}
+import org.apache.james.jmap.core.Invocation.{Arguments, MethodName}
+import org.apache.james.jmap.core.{Capability, CapabilityProperties, ClientId, Id, Invocation, ServerId, SetError}
+import org.apache.james.jmap.json.ResponseSerializer
+import org.apache.james.jmap.method.{InvocationWithContext, Method, MethodRequiringAccountId}
+import org.apache.james.jmap.routes.SessionSupplier
+import org.apache.james.mailbox.MailboxSession
+import org.apache.james.metrics.api.MetricFactory
+import play.api.libs.json.{JsError, JsObject, JsSuccess, Json}
+import reactor.core.scala.publisher.SMono
+
+import javax.inject.Inject
+
+case object KeystoreCapabilityProperties extends CapabilityProperties {
+  override def jsonify(): JsObject = Json.obj()
+}
+
+case object KeystoreCapability extends Capability {
+  val properties: CapabilityProperties = KeystoreCapabilityProperties
+  val identifier: CapabilityIdentifier = LINAGORA_PGP
+}
+
+class KeystoreCapabilitiesModule extends AbstractModule {
+  @ProvidesIntoSet
+  private def capability(): Capability = KeystoreCapability
+}
+
+class KeystoreSetMethodModule extends AbstractModule {
```

Comment:
You do not bind KeystoreCapability in the module :-p

Reply:
The submarine effect I guess...

Reply:
No I think I do? Below line:
```
install(new KeystoreCapabilitiesModule())
```
???

Reply:
Ok but I still miss the test.